share/completions/zypper.fish

```diff
@@ -20,7 +21,7 @@ function __fish_zypper_cmd_in_array
     return 1
 end
 
-function __fish_zypper_no_subcommand
+function __fish_zypper_no_command
```

Comment:
Please no. It's a subcommand *to zypper*, the function can be named `no_subcommand`.

Reply:
zypper has a feature named "subcommand" that allow standalone executables (user's script, for example) to be run as zypper subcommands, which this commit tries to complete. Isn't that confusing with same names?

Reply:
FYI:
```
$ zypper subcommand
<subcommand> [--command-options] [arguments]

Zypper subcommands are standalone executables that live in the
zypper_execdir ('/usr/lib/zypper/commands').

For subcommands zypper provides a wrapper that knows where the
subcommands live, and runs them by passing command-line arguments
to them.

If a subcommand is not found in the zypper_execdir, the wrapper
will look in the rest of your $PATH for it. Thus, it's possible
to write local zypper extensions that don't live in system space.

Using zypper global-options together with subcommands, as well as
executing subcommands in 'zypper shell' is currently not supported.

Available zypper subcommands in '/usr/lib/zypper/commands'

      appstream-cache       <No manual entry for zypper-appstream-cache>
      lifecycle             products and packages lifecycle information


Zypper subcommands available from elsewhere on your $PATH

      autoremove            <No manual entry for zypper-autoremove>
      log                   Zypper logfile reader


Type 'zypper help <subcommand>' to get subcommand-specific help if available.
```

Reply:
Okay, but in *fish*, a "subcommand" is something like the "log" in `git log` or the "install" in `zypper install`.

Which is reflected in helper functions like `__fish_seen_subcommand_from`.

Reply:
OK, I get it. So any ideas about naming here? Or shall I keep the confusing names?

Reply:
Just keep the old names, please.